content/docs/command-reference/get-url.md

```diff
@@ -34,23 +33,33 @@ directory will be placed inside.
 
 DVC supports several types of (local or) remote locations (protocols):
 
-| Type    | Description    | `url` format                               |
-| ------- | -------------- | ------------------------------------------ |
-| `local` | Local path     | `/path/to/local/data`                      |
-| `s3`    | Amazon S3      | `s3://mybucket/data`                       |
-| `gs`    | Google Storage | `gs://mybucket/data`                       |
-| `ssh`   | SSH server     | `ssh://user@example.com/path/to/data`      |
-| `hdfs`  | HDFS to file\* | `hdfs://user@example.com/path/to/data.csv` |
-| `http`  | HTTP to file\* | `https://example.com/path/to/data.csv`     |
+| Type     | Description                  | `url` format example                                       |
+| -------- | ---------------------------- | ---------------------------------------------------------- |
+| `s3`     | Amazon S3                    | `s3://bucket/data`                                         |
+| `azure`  | Microsoft Azure Blob Storage | `azure://container/data`                                   |
+| `gdrive` | Google Drive                 | `gdrive://<folder-id>/data`                                |
+| `gs`     | Google Cloud Storage         | `gs://bucket/data`                                         |
+| `ssh`    | SSH server                   | `ssh://user@example.com/path/to/data`                      |
+| `hdfs`   | HDFS to file\*               | `hdfs://user@example.com/path/to/data.csv`                 |
+| `http`   | HTTP to file\*               | `https://example.com/path/to/data.csv`                     |
+| `webdav` | WebDav to file\*             | `webdavs://example.com/public.php/webdav/path/to/data.csv` |
+| `local`  | Local path                   | `/path/to/local/data`                                      |
```

Comment:
Reordered list same as in remote add.modify refs. and added missing types (except OSS — not sure it's worth it).

Reply:
+ improved the way notes about this list are described (below: lines 54+)

Reply:
Same in import-url.